static/src/stylesheets/module/crosswords/_layout.scss

```diff
@@ -105,6 +105,7 @@
     height: 120px;
     background: linear-gradient(to bottom, rgba(255, 255, 255, 0), #ffffff);
     display: none;
+    pointer-events: none;
```

Comment:
Annnnd now I know how to fix @paperboyo's  [problem with not being able to click](https://github.com/guardian/dotcom-rendering/issues/1484) on links near drop caps. Thanks!

Reply:
:-) Will it still be possible to select the word including first letter (drop cap)? Not a hugie, but would be nicely pedantic.

Reply:
Yes

![2020-05-13 17 16 25](https://user-images.githubusercontent.com/1336821/81838064-8b401d80-953d-11ea-936d-8c4f4a830427.gif)